nilearn/glm/first_level/first_level.py

```diff
@@ -1060,6 +1034,249 @@ def first_level_from_bids(dataset_path, task_label, space_label=None,
                                  (len(events), len(imgs)))
             confounds = [pd.read_csv(c, sep='\t', index_col=None)
                          for c in confounds]
+
             models_confounds.append(confounds)
 
     return models, models_run_imgs, models_events, models_confounds
+
+
+def _bids_validate_arguments(dataset_path: str,
+                             task_label: str,
+                             space_label: str | None,
+                             img_filters: list[Tuple[str, str]],
```

Comment:
Yup I was getting errors in CI because of that: 
https://github.com/nilearn/nilearn/actions/runs/4251874122/jobs/7394767967#step:7:25

Fixed it by importing future annotations  
https://github.com/nilearn/nilearn/pull/3525/commits/afce52f0b71d53ab5887ce7bd3c4ae551223fe1b

Python 3.7 in CI was OK after that:
https://github.com/nilearn/nilearn/actions/runs/4251899059/jobs/7394830566

Reply:
Cool. I didn't know they had retro'd it for 3.7. Thanks.
Then I suppose instead of `Tuple`, we can use `tuple` here and everywhere else. 😁

Reply:
Ha yeah true. I should update that.